var/spack/repos/builtin/packages/kaldi/package.py

```diff
@@ -43,7 +44,8 @@ class Kaldi(Package):  # Does not use Autotools
     depends_on("cub", when="@2019-07-29:^cuda@:10")
 
     patch("openfst-1.4.1.patch", when="@2015-10-07")
-    patch("0001_CMakeLists_txt.patch", when="+cuda@11:")
+    patch("0001_CMakeLists_txt.patch", when="@:2021-11-16")
+    patch("0001-Fix-CUDA_INCLUDE_DIRS.patch", when="@2024-01-31:")
```

Comment:
Can you add a brief comment above each `patch` directive explaining:
1. What is the effect of the patch
2. Why it is needed
3. Where it was taken from

?

Reply:
The new patch 0001-Fix-CUDA_INCLUDE_DIRS.patch is just an adjustment of the previous patch 0001_CMakeLists_txt.patch which doesn't apply anymore for new kaldi versions. I wrote this trivial patch.

Reply:
@rfehren Sure, the request was for adding it in a comment above the patch directive. It helps when reading the recipe, so people can get a sense what the patches are doing without having to open/doenload and inspect each file.